cypress/integration/e2e/table/update-workhours-info.spec.js

```diff
@@ -0,0 +1,285 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/// <reference types="cypress" />
+
+const nurseInitialWorkHours = { required: 136, actual: 164, overtime: 28 };
+const babysitterInitialWorkHours = { required: 80, actual: 108, overtime: 28 };
```

Comment:
Dobrze rozumiem że to są dane dla konkretnej opiekunki i konkretnej pilęgniarki?
Jeżeli tak, to dodałbym w tym miejscu też indeks tego pracownika, bo może być tak, że ktoś kiedyś będzie zmieniać logikę i testy i wtedy aktualizacja tych godzin będzie trochę niewygodna, bo niewiadomo dla jakiego pracownika powinny być zauktualizowane.

Reply:
Tak, do testów idą dane 1 pielęgniarki i 1 opiekunki a dostawanie się do ich indeksów jest zrobione poprzez workerTypeToIdx które jest od razu po niżej, wydaje mi się to wystarczająco czytelne.

Reply:
Dobra, nie zauważyłem tego